charts/vaultwarden/templates/ingress.yaml

```diff
@@ -1,3 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: customer-headers-configmap
+data:
+  Request-Id: $req_id
```

Comment:
According to the documentation [here](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#custom-headers), you should also add `global-allowed-response-headers` to the ConfigMap.

This is the example I found:

```
global-allowed-response-headers: Referrer-Policy,Strict-Transport-Security,X-Content-Type-Options,X-Frame-Options,Content-Security-Policy
```